src/main/resources/swagger/agora.yaml

```diff
@@ -1406,6 +1406,52 @@ paths:
             - openid
             - email
             - profile
+  /ga4gh/tools/{id}/versions/{version-id}/{type}/descriptor:
+    get:
+      summary: Get the tool descriptor (CWL/WDL) for the specified tool.
+      description: Returns the CWL or WDL descriptor for the specified tool.
+      tags:
+        - GA4GH
+      parameters:
+        - name: type
+          required: true
+          in: path
+          description: >
+            The output type of the descriptor. If not specified it is up to the underlying
+            implementation to determine which output type to return. Plain types return the
+            bare descriptor while the "non-plain" types return a descriptor wrapped
+            with metadata.
+            In Agora, only WDL and plain-WDL are supported.
+          type: string
+          enum:
+            - CWL
+            - WDL
+            - plain-CWL
+            - plain-WDL
+        - name: id
+          in: path
+          description: >
```

Comment:
curiosity Q: how is `>` different to `|`?

Reply:
Good answer [here](https://stackoverflow.com/questions/15540635/what-is-the-use-of-pipe-symbol-in-yaml/18708156#18708156) ... `|` preserves newlines, `>` ignores them ... except for indented blocks.